vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Supplying usage for ports
```

Comment:
What does Supplying usage for ports mean? As I read the article, it seems like a more appropriate title is something like "Provide usage documentation for ports"